### PR TITLE
[FIX] Analysis wizard: Rename field "write" to "write_files"

### PR DESCRIPTION
**Pull request by alhashash**
To prevent error when calling the orm "write()" method on the wizard model as it calls the "write" field descriptor instead which raises `AttributeError: 'openupgrade.analysis.wizard' object has no attribute '_ids'`

### CODE REVIEW

File: openerp/addons/openupgrade_records/model/analysis_wizard.py [hunk 1]
```diff
@@ -55,14 +55,14 @@ class openupgrade_analysis_wizard(TransientModel):
...
-        'write': fields.boolean(
+        'write_files': fields.boolean(
             'Write files',
             help='Write analysis files to the module directories'
             ),
         }
     _defaults = {
         'state': lambda *a: 'init',
-        'write': lambda *a: True,
+        'write_files': lambda *a: True,
```
Comment (pedrobaeza) [inline openerp/addons/openupgrade_records/model/analysis_wizard.py]: You can put only `'write_files':True,`

Comment (alhashash) [inline openerp/addons/openupgrade_records/model/analysis_wizard.py]: Sure, for both fields, but I just made the minimum amount of change to fix the issue.

**Comment by pedrobaeza** [comment]
:+1: 

Thanks.

**Comment by legalsylvain** [comment]
:+1: Thanks

**Pull request merged**